src/Component/ContactUsForm.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState} from 'react'
+import '../Style/index.css'
+import axios from 'axios'
+
+// to reset a form
+const INITIAL_STATE = {
+    name:'',
+    email:'',
+    subject:'',
+    message:'',
+}
+
+const ContactUsForm = () => {
+    const [form, setForm] = React.useState({INITIAL_STATE}) 
+    const [formErrors, setformErrors] = useState({})
+   // const [isSubmit, setIsSubmit] = useState(false)
+    const [loading, setLoading] = useState(false);
+
+    const handleChange = (event) => {
+        setForm({...form,
+        [event.target.id]: event.target.value,
+    });
+    }
+    
+    const handleSubmit = (event) => {
+        event.preventDefault();
+        setForm(INITIAL_STATE);
+        setformErrors(validate(form))
+        setLoading(true)
+        axios.post('https://my-json-server.typicode.com/tundeojediran/contacts-api-server/inquiries', {form})
+        .then(response => console.log(response))
+        .catch(err => console.log (err))
+    };
```

Comment:
You need to reset your loading state in this axios. A rewrite might look like this. 
```

axios.post('https://my-json-server.typicode.com/tundeojediran/contacts-api-server/inquiries', {form})
        .then((response) => {
        setLoading(false);
        setformErrors({});
        })
        .catch(err => {
         setLoading(false);
        })
    };
```

Compare the difference between these and yours